src/main/webapp/src/app/recommendation-item/recommendation-item.component.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <mat-card class="mat-elevation-24">
```

Comment:
iirc the cards on the main page had class mat-elevation-z2.
Should this be the same to keep a consistent look or do you think it's ok for the recommendations page to have a different aesthetic?

Reply:
Changed to  mat-elevation-z2